presto-parser/src/main/java/com/facebook/presto/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -484,13 +482,30 @@ protected R visitGroupBy(GroupBy node, C context)
     }
 
     @Override
-    protected R visitGroupingElement(GroupingElement node, C context)
+    protected R visitCube(Cube node, C context)
+    {
+        return null;
```

Comment:
please add a comment (here, in rollup and in grouping sets)

Reply:
I’m not sure what the comment would say. There’s nothing special about this. The node doesn’t have any children to recurse into — it’s a leaf.